packages/thirdweb/src/exports/extensions/permissions.ts

```diff
@@ -57,3 +57,8 @@ export {
   getAllRoleMembers,
   type GetAllRoleMembersParams,
 } from "../../extensions/permissions/read/getAllMembers.js";
+
+// --------------------------------------------------------
+// Utils
+// --------------------------------------------------------
+export { roleMap, getRoleHash } from "../../extensions/permissions/utils.js";
```

Comment:
since we're exporting these now please add doc comments to them so when they show up in docs they have some information with them